source/include/FreeRTOS_Routing.h

```diff
@@ -54,6 +54,10 @@
 /* Return true as long as the LinkStatus on the PHY is present. */
     typedef BaseType_t ( * GetPhyLinkStatusFunction_t ) ( struct xNetworkInterface * pxDescriptor );
 
+/* Functions that manipulate what MAC addresses are received by this interface */
+    typedef void ( * NetworkInterfaceMACFilterFunction_t ) ( struct xNetworkInterface * pxDescriptor,
```

Comment:
I think you mean `pxNetworkInterface` in stead of `pxDescriptor`?

Reply:
Oops, looking at the original source ai See that this mistake is made 4 times.
I propose to replace:
~~~
struct xNetworkInterface * pxDescriptor;
~~~
with:
~~~
struct xNetworkInterface * pNetworkInterface;
~~~

Reply:
Do you mean replace it for just the MAC filtering functions or all of them including
`NetworkInterfaceInitialiseFunction_t`
`NetworkInterfaceOutputFunction_t`
`GetPhyLinkStatusFunction_t`